src/Makefile.am

```diff
@@ -250,10 +250,13 @@ BITCOIN_CORE_H = \
   wallet/crypter.h \
   wallet/db.h \
   wallet/dump.h \
+  wallet/externalsigner.h \
+  wallet/external_signer_scriptpubkeyman.h \
```

Comment:
nit: I'd prefer these to be consistent in naming; either
```
  wallet/externalsigner.h \
  wallet/externalsigner_scriptpubkeyman.h \
```
or
```
  wallet/external_signer.h \
  wallet/external_signer_scriptpubkeyman.h \
```
(same for the associated `.cpp`, of course)

Reply:
I'll go for the second option, because it's more readable.